docs/stylesheets/extra.css

```diff
@@ -9,27 +13,28 @@
 .big-font {
   font-size: 30px;
 }
-
 .secondary-color {
   color: #9dc1e0;
 }
 strong {
   color: #5480cb;
 }
-.strong-extra {
-  color: #5480cb;
-}
+
 .md-ellipsis:hover {
   color: #c99300;
 }
-
 .link-home {
   color: #c99300 !important;
 }
-
+.link-home:hover {
+  color: #5480cb !important;
+}
 .md-typeset a {
   color: #5480cb;
 }
```

Comment:
I know this isn't part of the PR but noticed that the strong-faced font & icons look quite light in Light mode! 
<img width="792" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/1db3ba8b-861f-47f3-b210-0309cf0a4b75">

Also, this blue color feels like it should be a link. I think that's what visitors might be used to also?

Reply:
I agree, ticketing this. I was unable to figure a fix